web-www/src/components/ConnectSection.tsx

```diff
@@ -78,17 +78,19 @@ const IconLabel = ({
         <Box
           sx={theme => ({
             bgcolor: 'secondary.main',
+            background:
+              'linear-gradient(201.8deg, #4FB573 14.67%, #B9E1C7 97.14%);',
```

Comment:
I'm guessing this can't (should not ?) be added to the theme.

Reply:
it’s a gradient specific to round buttons like this, so probably not worth having in the theme. 

One pattern I like to do in other apps is something like a `styles.constants.ts` file, and could have something like 
```ts
GRADIENTS = {
  round: ''linear-gradient(201.8deg, #4FB573 14.67%, #B9E1C7 97.14%);’'
  button: ..etc
  }
```

but I didn’t want to make that sort of change to the style of the codebase without talking. Doing it here seems more in line with current patterns (though I could extract it to a shared compoonent, but figured we would do that if we reuse anywhere)